scipy/stats/_continuous_distns.py

```diff
@@ -5670,6 +5680,10 @@ def _stats(self, nu):
         g2 /= nu*mu2**2.0
         return mu, mu2, g1, g2
 
+    def _rvs(self, nu, size=None, random_state=None):
+        # this relationship can be found in [1] or by a direct calculation
+        return np.sqrt(random_state.standard_gamma(nu, size=size) / nu)
```

Comment:
We don't pass any scale? Shouldn't this read `np.sqrt(random_state.standard_gamma(nu, size=size) / nu * scale)`?

Reply:
> We don't pass any scale?

I think the relationship between Wikipedia's parametrization and SciPy's parameterization is: `nakagami(m, omega) = stats.nakagami(m, loc=0, scale=np.sqrt(omega))`. So, the rvs can easily be scaled by passing `loc=0, scale=np.sqrt(omega)`: `rvs_scaled = nakagami.rvs(m, loc=0, scale=np.sqrt(omega))`. A note in the documentation explaining the relation between these parameterizations would help but would be a bit unrelated to changes here. Maybe, tackle it in a different PR?